File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/prepackaged/Dog.java
package com.raizlabs.android.dbflow.test.prepackaged;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

@Table(database = TestDB.class)
public class Dog extends BaseModel {

    @PrimaryKey
    int id;

    @Column
    String breed;

    @Column
    String color;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/prepackaged/Dog.kt
package com.raizlabs.android.dbflow.prepackaged

import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

@Table(database = TestDB::class)
class Dog : BaseModel() {

    @PrimaryKey
    var id: Int = 0

    @Column
    var breed: String? = null

    @Column
    var color: String? = null
}
